gobuilds.sh

```diff
@@ -3,6 +3,8 @@ set -eux
 
 # This one hasn't been migrated to the go test yet.
 (cd ./test/requestconflict && ./test.sh)
-
-(cd ./src/cmd/makebb && GO111MODULE=on go build .)
-(cd ./test && go test --makebb=../src/cmd/makebb/makebb -v)
```

Comment:
tests are not shell files anymore, they're now a Go test in ./test directory -- add your new kind of test there instead of as a shell file